ccaas_builder/go.mod

```diff
@@ -0,0 +1,5 @@
+module hyperledgendary/fabric-ext-builder/v2
+
+go 1.16
```

Comment:
mmm I thought it had all been pushed up to 1.16.. @denyeart ?

Reply:
The project currently uses Go 1.16.
As far as I can tell, the value in the go directive here doesn't really matter since there have been no incompatible language changes, and therefore the Fabric go.mod still says 1.14. For new components such as this one, I would utilize the current version which is go 1.16.